jmx/plugins/org.jboss.tools.jmx.jolokia/src/org/jboss/tools/jmx/jolokia/internal/connection/JolokiaMBeanServerConnection.java

```diff
@@ -293,15 +336,15 @@ public void unregisterMBean(ObjectName name)
 	
 	@Override
 	public ObjectInstance createMBean(String className, ObjectName name)
-			throws ReflectionException, InstanceAlreadyExistsException, MBeanRegistrationException, MBeanException,
+			throws ReflectionException, InstanceAlreadyExistsException, MBeanException,
```

Comment:
it was reported as a Code smell because MbeanRegistration is a subclass of one of the other Exception (surely MBeanException)